stack.yaml

```diff
@@ -22,6 +22,10 @@ packages:
 - explorer
 
 - location:
+    git: https://github.com/serokell/log-warper.git
+    # master, but we should switch to stackage when 1.2.2 will be on it
```

Comment:
@volhovM can you explain what are we waiting for? Previously it was enough for a package to be on hackage. What changed?

Reply:
I'm not sure it ever was as simple as you say. It seems that i'm wrong after all and we don't use `nixpkgs`' hackage clone to get packages we need so putting log-warper from hackage will work. I will switch it now.